### PR TITLE
update swap related description before installation

### PR DESCRIPTION
**Pull request by pacoxu**
xref https://github.com/kubernetes/kubeadm/issues/2563#issuecomment-1696074991

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ see the [Creating a cluster with kubeadm](/docs/setup/production-environment/too
...
 * Unique hostname, MAC address, and product_uuid for every node. See [here](#verify-mac-address) for more details.
 * Certain ports are open on your machines. See [here](#check-required-ports) for more details.
-* Swap disabled. You **MUST** disable swap in order for the kubelet to work properly.
-    * For example, `sudo swapoff -a` will disable swapping temporarily. To make this change persistent across reboots, make sure swap is disabled in config files like `/etc/fstab`, `systemd.swap`, depending how it was configured on your system.
+* Swap configuration. The default behavior of a kubelet was to fail to start if swap memory was detected on a node. Swap has been supported since v1.22. And since v1.28, Swap is supported for cgroup v2 only; the NodeSwap feature gate of the kubelet is beta but disabled by default.
+    * You **MUST** disable swap if the kubelet is not properly configured to use swap. For example, `sudo swapoff -a` will disable swapping temporarily. To make this change persistent across reboots, make sure swap is disabled in config files like `/etc/fstab`, `systemd.swap`, depending how it was configured on your system.
```
Comment (tengqm) [inline content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:30]: Please 1) wrap the long lines manually for ease of change tracking and review, and localization; 2) avoid indentation by 4 spaces which may confuse the Markdown parser.

Comment (pacoxu) [reply content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md]: Updated. Thanks.

**Review by neolit123 — COMMENTED** [review]
thanks
this update has the correct details, but it forces us to update this description again after the fg is ga...and possibly another time.

can we just say "swap is condotionally supported. please read more here..." and link to some external page?

**Comment by pacoxu** [comment]
> can we just say "swap is condotionally supported. please read more here..." and link to some external page?

Probably link to https://kubernetes.io/blog/2023/08/24/swap-linux-beta/.

**Comment by neolit123** [comment]
> > can we just say "swap is condotionally supported. please read more here..." and link to some external page?
> 
> Probably link to https://kubernetes.io/blog/2023/08/24/swap-linux-beta/.

+1
let's see if sig docs agree

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**